index.js

```diff
@@ -6,14 +6,14 @@ function noop () {
   return undefined
 }
 
-function simpleBody (context) {
-  return context.reply.request.body.query
+function simpleBody (context, body) {
+  return body?.query ?? context.reply.request.body.query
```

Comment:
The CI output shows that the coverage is decreased without failing the CI:

<img width="702" alt="image" src="https://github.com/Eomm/mercurius-logging/assets/11404065/71ba3157-8b23-4c98-aee7-87931ab03f83">

The body is always set

```suggest
return body.query
```